ui/jstests/test-learning-resource.jsx

```diff
@@ -220,38 +231,88 @@ define(['QUnit', 'jquery', 'react', 'lodash', 'learning_resources',
           // TestUtils.Simulate.change only simulates a change event,
           // we need to update the value first ourselves
           $vocabSelect.val("prerequisite").trigger('change');
-          React.addons.TestUtils.Simulate.change($vocabSelect);
           component.forceUpdate(function() {
             assert.equal($vocabOptions[0].selected, false);
             assert.equal($vocabOptions[1].selected, true);
-            // on selection of the second vocabulary, make sure the term selector updates.
+            // re-fetch the selects to get the prerequisite one
+            $allSelects = $node.find("#vocabularies select");
             var termsSelect = $allSelects[1];
             var $termsOptions = $(termsSelect).find("option");
             assert.equal($termsOptions.size(), 2);
-            assert.equal($termsOptions[0].selected, false);
+            assert.equal($termsOptions[0].selected, true);
             assert.equal($termsOptions[1].selected, false);
-            // the second vocabulary can be a multi select
-            $(termsSelect)
-              .val(["hard", "easy"])
-              .trigger('change');
-            React.addons.TestUtils.Simulate.change(termsSelect);
+            assert.equal($(termsSelect).val(), "required");
+
+            // remove selection for this vocabulary to not interfer to the rest of the test
+            $(termsSelect).val("").trigger('change');
             component.forceUpdate(function() {
-              assert.equal($termsOptions[0].selected, true);
-              assert.equal($termsOptions[1].selected, true);
-              // be sure that the state reflects the selection
-              var terms = _.map(component.state.vocabulariesAndTerms,
-                function (tuple) {
-                  return tuple.selectedTerms;
+              // Switch to difficulty
+              $vocabSelect.val("difficulty").trigger('change');
+              component.forceUpdate(function() {
+                // re-fetch the selects to get the prerequisite one
```

Comment:
It seems like you are refetching to get the difficulty selects not the prerequisite